# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: docshell/base/BrowsingContext.h
# Commit: 95b350572bbf
# Full Hash: 95b350572bbf079fdf6613b5a721c6ee5023fc6e
# Author: Kashav Madan <kmadan@mozilla.com>
# Date: 2020-06-23 16:15:06
# Regressor Bug: 1646563
# File Overlap Count: 1
# Description:
#   Bug 1646563 - Remove BrowserId from nsFrameLoaderOwner, r=nika
#   
#   We can just use BrowsingContext::BrowserId directly, so it's unnecessary to have
#   the field on nsFrameLoaderOwner as well.
#   
# ==============================================================================

diff -r 4eab38aba3fe -r 95b350572bbf docshell/base/BrowsingContext.h
--- a/docshell/base/BrowsingContext.h	Mon Jun 22 21:30:31 2020 +0000
+++ b/docshell/base/BrowsingContext.h	Mon Jun 22 21:46:03 2020 +0000
@@ -205,7 +205,7 @@
   // DocShell, BrowserParent, or BrowserBridgeChild.
   static already_AddRefed<BrowsingContext> CreateDetached(
       nsGlobalWindowInner* aParent, BrowsingContext* aOpener,
-      const nsAString& aName, Type aType, uint64_t aBrowserId);
+      const nsAString& aName, Type aType);
 
   void EnsureAttached();
 